### PR TITLE
Fix compilation errors due to changes in o3de rendering API

### PR DESCRIPTION
**Pull request by michalpelka**
Signed-off-by: Michał Pełka <michalpelka@gmail.com>

### CODE REVIEW

File: Code/Source/Camera/CameraSensor.cpp [hunk 1]
```diff
@@ -132,7 +132,7 @@ namespace ROS2
         AZ::Transform inverse = cameraPose.GetInverse();
         m_view->SetWorldToViewMatrix(AZ::Matrix4x4::CreateFromQuaternionAndTranslation(inverse.GetRotation(), inverse.GetTranslation()));
 
-        size_t userId = AZ::Render::FrameCaptureRequests::s_InvalidFrameCaptureId;
+        size_t userId = AZ::Render::InvalidFrameCaptureId;
```
Comment (piotr-zyskowski-rai) [inline Code/Source/Camera/CameraSensor.cpp]:
```suggestion
        AZ::Render::FrameCaptureId captureId = AZ::Render::InvalidFrameCaptureId;
```

**Review by adamdbrw — APPROVED** [review]

**Comment by adamdbrw** [comment]
@piotr-zyskowski-rai resolve conflicts please and we can merge it

**Comment by piotr-zyskowski-rai** [comment]
@adamdbrw It will not compile in its current state. I will fix it and resolve conflicts.

**Comment by piotr-zyskowski-rai** [comment]
@adamdbrw Now it should be fine

**Review by adamdbrw — APPROVED** [review]

**Pull request merged**